src/components/settings/ARDownloadList.js

```diff
@@ -0,0 +1,133 @@
+import * as FileSystem from 'expo-file-system';
+import React, { useEffect, useState } from 'react';
+import { ActivityIndicator, Alert, FlatList, StyleSheet } from 'react-native';
+import { Divider } from 'react-native-elements';
+
+import { colors, device, normalize, texts } from '../../config';
+import {
+  checkDownloadedData,
+  deleteAllData,
+  downloadAllData,
+  dummyData,
+  formatSize
+} from '../../helpers';
+import { ARDownloadListItem } from '../ARDownloadListItem';
+import { Button } from '../Button';
+import { BoldText, RegularText } from '../Text';
+import { Touchable } from '../Touchable';
+import { Wrapper, WrapperRow } from '../Wrapper';
+
+const downloadAllDataAlert = (downloadAll) =>
+  Alert.alert(
+    texts.settingsTitles.arListLayouts.alertTitle,
+    texts.settingsTitles.arListLayouts.allDownloadAlertMessage,
+    [
+      {
+        text: texts.settingsTitles.arListLayouts.downloadAlertButtonText,
+        onPress: downloadAll,
+        style: 'default'
+      },
+      { text: texts.settingsTitles.arListLayouts.cancel, style: 'cancel' }
+    ]
+  );
+
+const deleteAllDataAlert = (deleteAll) =>
+  Alert.alert(
+    texts.settingsTitles.arListLayouts.alertTitle,
+    texts.settingsTitles.arListLayouts.allDeleteAlertMessage,
+    [
+      {
+        text: texts.settingsTitles.arListLayouts.deleteAlertButtonText,
+        onPress: deleteAll,
+        style: 'destructive'
+      },
+      { text: texts.settingsTitles.arListLayouts.cancel, style: 'cancel' }
+    ]
+  );
+
+const renderItem = ({ data, index, item, setData }) => (
+  <ARDownloadListItem item={item} index={index} data={data} setData={setData} />
+);
+
+export const ARDownloadList = () => {
+  const [data, setData] = useState(dummyData);
+  const [loading, setLoading] = useState(false);
+  const [freeSize, setFreeSize] = useState(0);
+
+  useEffect(() => {
+    checkFreeStorage();
+  }, [data]);
+
+  useEffect(() => {
+    checkinData();
+  }, []);
+
+  const checkinData = async () => {
+    setLoading(false);
+    const { newDownloadedData } = await checkDownloadedData({ downloadableData: data });
+    setData(newDownloadedData);
+    setLoading(false);
+  };
+
+  const checkFreeStorage = async () => {
+    const storage = await FileSystem.getFreeDiskStorageAsync();
+    setFreeSize(formatSize(storage));
+  };
+
+  const downloadAll = async () => {
+    downloadAllData({ downloadableData: data, setDownloadableData: setData });
+  };
+
+  const deleteAll = async () => {
+    deleteAllData({ downloadableData: data, setDownloadableData: setData });
+  };
+
+  if (loading) return <ActivityIndicator size="small" />;
+
+  return (
+    <>
+      <Wrapper style={styles.wrapper}>
+        <WrapperRow spaceBetween>
+          <BoldText>{texts.settingsTitles.arListLayouts.arListTitle}</BoldText>
+        </WrapperRow>
+      </Wrapper>
+
+      <Divider style={styles.divider} />
```

Comment:
what do you mean exactly? the design we had for headline/titles is the section header?

Reply:
I did it this way by taking the design in the Figma file as an example, but now I have updated it.

Reply:
i understand. but the Figma is explicitly for a community, not the master repo. the section header differs between communities. so later for that specific example for herzberg it will be "just black on white".

Reply:
I think this matter is closed.